content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -115,29 +115,43 @@ documentation for the plugins about what port(s) those need.
 To run containers in Pods, Kubernetes uses a
 {{< glossary_tooltip term_id="container-runtime" text="container runtime" >}}.
 
+{{< tabs name="container_runtime" >}}
+{{% tab name="Linux nodes" %}}
+
 By default, Kubernetes uses the
 {{< glossary_tooltip term_id="cri" text="Container Runtime Interface">}} (CRI)
 to interface with your chosen container runtime.
 
-kubeadm automatically tries to detect the container runtime on Linux nodes
-by scanning through a list of well known domain sockets. The detectable runtimes and the
-socket paths that are tried are listed in the following table:
+kubeadm automatically tries to detect the container runtime
```

Comment:
nit: line 121: Is this statement true for all OSes (the statement appears only in the Linux tab)?
```
By default, Kubernetes uses the
{{< glossary_tooltip term_id="cri" text="Container Runtime Interface">}} (CRI)
to interface with your chosen container runtime.
```

Reply:
Hmm, good point. Given the prerequisites it might be easier to drop:
> On non-Linux nodes the container runtime used by default is Docker.

(text as per original).